File: primer-curso/programacion/unidad-01-introduccion-programacion/python/Calculadora.py
opcion = input("Inserte el tipo de operacion que quieres realizar (suma, resta, multiplicacion, division): ").lower()

#Comprobando valores iguales tendremos que utilizar la orden or mientras que comparando que 
# no sean iguales, utilizaremos and porque no podra ser ninguna de esas

if opcion != "suma" and opcion != "resta" and opcion != "multiplicacion" and opcion != "division":
	print("Operacion no valida")
	exit()

print("Continuar")

num1 = float(input("Numero 1º: "))
num2 = float(input("Numero 2º: "))


if opcion == "suma":
	resultado = num1 + num2
	print(f"{num1} + {num2} = {resultado}")
elif opcion == "resta":
	resultado = num1 - num2
	print(f"{num1} - {num2} = {resultado}")
elif opcion == "division":
	if num2 == 0:
		print("No se puede dividir entre cero")
	else:
		resultado = num1/num2 
		print(f"{num1} / {num2} = {resultado}")
elif opcion == "multiplicacion":
	resultado = num1 * num2
	print(f"{num1} * {num2} = {resultado}")

File: primer-curso/programacion/unidad-01-introduccion-programacion/java/calculadora.java
import java.util.Scanner;

public class calculadora{
	public static void main(String[] args){
		Scanner teclado = new Scanner(System.in);

		System.out.print("CALCULADORA DNS 1.0\n----------------\n\n1 - Suma\n2 - Resta\n3 - Multiplicacion\n4 - Division\n\nSeleccione el numero para ejecutar la operacion: ");
		int elegir = Integer.parseInt(teclado.nextLine());

		if(elegir <= 0 || elegir > 4){

			System.out.println("La operacion introducida no es valida");
			System.exit(0);
		}

		System.out.print("Introduzca el primero numero con el que calcularemos: ");
		int num1 = Integer.parseInt(teclado.nextLine());

		System.out.print("Introduzca el primero numero con el que calcularemos: ");
		int num2 = Integer.parseInt(teclado.nextLine());

		if (elegir == 1){
			System.out.println(num1 +" + " +num2+ " = " + (num1+num2));
		}
		else if (elegir == 2){
			System.out.println(num1 +" - " +num2 +" = " + (num1-num2));
		}
		else if (elegir == 3){
			System.out.println(num1 +" * " +num2 +" = " + (num1*num2));
		}
		else if (elegir == 4){
			if (num2 <= 0){
				System.out.println("No se puede dividir entre 0");
				System.exit(0);
			}
			else{
				System.out.println(num1 +" / " +num2 +" = " + (num1/num2));
			}
		}
	}
}